aim/api/resource.py

```diff
@@ -86,7 +89,6 @@ class Tenant(AciResourceBase):
 
     identity_attributes = ['name']
     other_attributes = ['display_name']
-    db_attributes = []
 
     _aci_mo_name = 'fvTenant'
     _tree_parent = None
```

Comment:
Northbound clients are not required to create the Tenant resource, but it wouldn't harm anything if you created one. I'll leave up to you to make a choice.